bundles/extensions/servlet-helpers/src/test/java/org/apache/sling/servlethelpers/MockSlingHttpServletRequestTest.java

```diff
@@ -88,6 +88,44 @@ public void testSession() {
     }
 
```

Comment:
A test for getRequestURL would be useful as well showing the default port behaviour